server/utils/logger.ts

```diff
@@ -166,8 +169,109 @@ if (config.get('logging.file.enabled')) {
     level: config.get('logging.file.level'),
     path: logPath,
   })
+}
+
+async function processStroomFiles() {
+  const stroomFolder = resolve(appRoot, config.get('logging.stroom.folder'))
+  const processingFolder = resolve(stroomFolder, 'processing')
+
+  const files = (
+    await fsPromise.readdir(stroomFolder, {
```

Comment:
This logic is repeated below for `processingFolder`, could spin out into a reusable function